### DISCUSSION TITLE
No kerning data in JSON file

### DISCUSSION THREAD

**Discussion by johan-overbye** [Chlumsky/msdf-atlas-gen#61] [Q&A] [answered]
First of all, I'm really happy to have found this tool. Thanks for making it!

I'd like to run the command-line tool and output a distance field texture atlas along with kerning distance info for all the glyph pairs.

Based on `-help`, this seems to be enabled by default:
```
  -nokerning
      Disables inclusion of kerning pair table in output files.
```

However when I run this command:
```
msdf-atlas-gen.exe -font CascadiaMono.ttf -imageout msdf.png -json msdf.json
```

...there is no kerning info in the resulting JSON file:
```
"kerning":[]
```

How do I get a "kerning pair table" in the JSON file?

**Comment by Chlumsky** [comment, marked as answer]
This project relies on FreeType to load kerning information. If it fails to do so, it will not be available. See also #4. If you find that FreeType does load this information but it isn't present in the JSON output, please report it as a bug.

**Reply by johan-overbye** [reply]
Thanks. It was just because the font I tested with is monospaced. Works as expected.
